app/views/application/_question_radio_buttons.html.erb

```diff
@@ -13,22 +13,27 @@
   "id": question,
   "novalidate": "true"
 ) do %>
+  <%= render "govuk_publishing_components/components/radio", {
+    heading: t("coronavirus_form.groups.#{group}.questions.#{question}.title"),
+    heading_caption: t("coronavirus_form.groups.#{group}.title", { default: false }),
+    is_page_heading: true,
+    name: question,
+    error_message: error_items(question),
+    items: t("coronavirus_form.groups.#{group}.questions.#{question}.options").map do |option|
+        {
+          value: option,
+          text: option,
+          id: "option_#{option.parameterize.underscore}",
+          checked: @form_responses[question.to_sym] == option,
+        }
+      end
+  } %>
+  <%= render "govuk_publishing_components/components/button", text: "Continue", margin_bottom: true %>
+<% end %>
 
-<%= render "govuk_publishing_components/components/radio", {
-  heading: t("coronavirus_form.groups.#{group}.questions.#{question}.title"),
-  heading_caption: t("coronavirus_form.groups.#{group}.title", { default: false }),
-  is_page_heading: true,
-  name: question,
-  error_message: error_items(question),
-  items: t("coronavirus_form.groups.#{group}.questions.#{question}.options").map do |option|
-      {
-        value: option,
-        text: option,
-        id: "option_#{option.parameterize.underscore}",
-        checked: @form_responses[question.to_sym] == option,
-      }
-    end
-} %>
-
-<%= render "govuk_publishing_components/components/button", text: "Continue", margin_bottom: true %>
+<% content_for :escape_link do %>
+  <%= render "components/escape-link", {
```

Comment:
Since this is stuck to the bottom of the window, is it worth thinking about where best to put this so that people who use screenreader would be aware of it? Would moving it closer to the start of the markup make it easier to find, whilst not affecting its visible layout?

Reply:
Given a screen reader will know if the question is sensible only after reaching the question I consider it's more useful to be able to continue down the page to reach the exit link than going backwards (which usually requires an extra key to be pressed). I wouldn't be too concerned about this as from observed research screen reader users know how to quickly close the current tab/window using shortcuts (sometimes predefined) compared to other users. Open to suggestions if we have more insights on this.

Reply:
@alex-ju yep - what you say makes sense. Thanks for sharing your insight on this 👍